code/API_definitions/location-retrieval.yaml

```diff
@@ -42,6 +45,13 @@ info:
 
     * **Last Location Time** : Last date and time when the device was localized.
 
+    * **Max Surface**: Maximum surface in square meters which is accepted by the client for the location retrieval.
+
+      * absence of `maxSurface` means that "any surface size" is acceptable for the client.
+      * API implementation could specify in the documentation the `maxSurface` minimum acceptable (for exemple minimun of 10000 square meters allowed).

```

Comment:
I'd like to propose a minor rewording here, and fixing some typos:
API implementation could specify the minimum acceptable `maxSurface`  in the documentation (for example a minimum of 10000 square meters are allowed).